hack/testing/prep-host

```diff
@@ -10,6 +10,10 @@ OS_ANSIBLE_REPO=${OS_ANSIBLE_REPO:-https://github.com/openshift/openshift-ansibl
 OS_ANSIBLE_BRANCH=${OS_ANSIBLE_BRANCH:-master}
 OS_ANSIBLE_DIR=$WORKDIR/openhift-ansible
 
+if [ -n PUDDLE_URL ]; then
```

Comment:
I deferred to RPM here because I assume that is ultimately what we prefer.  I needed it for the ruamel-yaml rpm anyway because the code does not match what is available... at least for now